core/src/main/java/dk/alexandra/fresco/framework/builder/numeric/field/BigIntegerFieldDefinition.java

```diff
@@ -0,0 +1,98 @@
+package dk.alexandra.fresco.framework.builder.numeric.field;
+
+import dk.alexandra.fresco.framework.util.StrictBitVector;
+import java.math.BigInteger;
+import java.util.List;
+
+/**
+ * Defines a field based on a modulus as a normal BigInteger.
```

Comment:
```suggestion
 * Defines a field based on a {@link BigIntegerModulus}
```

Reply:
modulus as a normal BigInteger, is not very descriptive of what kind of field this can represent.